airbyte-server/build.gradle

```diff
@@ -49,26 +45,17 @@ dependencies {
     implementation libs.airbyte.protocol
     implementation project(':airbyte-persistence:job-persistence')
 
-    implementation 'com.github.slugify:slugify:2.4'
-    implementation 'commons-cli:commons-cli:1.4'
+    implementation libs.slugify
     implementation libs.temporal.sdk
-    implementation 'org.apache.cxf:cxf-core:3.4.2'
-    implementation 'org.eclipse.jetty:jetty-server:9.4.31.v20200723'
-    implementation 'org.eclipse.jetty:jetty-servlet:9.4.31.v20200723'
-    implementation 'org.glassfish.jaxb:jaxb-runtime:3.0.2'
```

Comment:
all the removed deps were jersey related?

Reply:
@davinchia Correct.  Some were not even used by the code -- I assume some old stuff that were not removed when some code was previously removed.